main.js

```diff
@@ -15,82 +15,26 @@ import {
   TextInput,
   Button,
   Alert,
-  Image
+  Image,
 } from 'react-native';
+import { Router, Scene } from 'react-native-router-flux'
+
+import Home from './Home';
+import Login from './login';
+import Forgot from './Forgot';
```

Comment:
You should try to standardize your naming convention, so that you either capitalize all your .js files our you don't